src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -28,13 +29,19 @@ object ProjectBuilder {
         }
     }
 
-    val dockerfiles = templefile.services.zip(1024 until Int.MaxValue).map {
-      case ((name, service), port) =>
+    val dockerfiles = templefile.servicesWithPorts.map {
+      case (name, service, port) =>
         val dockerfileRoot     = DockerfileBuilder.createServiceDockerfile(name.toLowerCase, service, port)
         val dockerfileContents = DockerfileGenerator.generate(dockerfileRoot)
         (File(s"${name.toLowerCase}", "Dockerfile"), dockerfileContents)
     }
 
-    Project(databaseCreationScripts ++ dockerfiles)
+    val orchestrationRoot = OrchestrationBuilder.createServiceOrchestrationRoot(
```

Comment:
Just a consistency side note - in the `database` and `dockerfile` cases we map over the services in this file, whereas with Kubernetes we map over in the builder - any particular reason why?

Reply:
The kube gen acts on all services at once (This way shared file i.e kong get generated properly)